domain/consensus/processes/pruningmanager/pruningmanager.go

```diff
@@ -296,6 +295,10 @@ func (pm *pruningManager) deletePastBlocks(stagingArea *model.StagingArea, pruni
 	return nil
 }
 
+func (pm *pruningManager) isVirtualGenesisOnlyParent(parents []*externalapi.DomainHash) bool {
```

Comment:
We have 2 of these functions.
Consider moving to `consensus/utils` or something.